docs/root/faq/concurrency_lb.rst

```diff
@@ -0,0 +1,12 @@
+Why doesn't RR load balancing appear to be even?
+================================================
+
+Envoy utilizes a siloed :ref:`threading model <arch_overview_threading>`. This means that worker
+threads and the load balancers that run on them do not coordinate with each other. When utilizing
+load balancing policies such as :ref:`round robin <arch_overview_load_balancing_types_round_robin>`,
+it may thus appear that load balancing is not working properly when using multiple workers. The
+:option:`--concurrency` option can be used to adjust the number of workers if desired.
+
+Note that the siloed execution model is the same reason that multiple HTTP/2 connections may be
+established to each upstream: :ref:`connection pools <arch_overview_conn_pool>` are not shared
```

Comment:
There is a grammar glitch in this sentence.

Reply:
Can you suggest a change? It reads OK to me.

Reply:
I think it might read a little better with "the siloed execution model is also the reason why multiple HTTP/2 connections"

Reply:
I think just replacing the `:` with a `;` would be fine (I actually missed it the first time around). I also like @mpuncel's suggestion.